h2o-core/src/main/java/water/util/FrameUtils.java

```diff
@@ -1253,6 +1252,8 @@ public void map(Chunk[] chunks, NewChunk[] newChunks) {
       for (int rowIndex=0; rowIndex<chkLen; rowIndex++) {
         String cName = chunks[0].atStr(tempStr, rowIndex).toString();
         int trainColNumber = trainColNames.indexOf(cName);
+        if (trainColNumber < 0 && "intercept".equals(cName))
```

Comment:
I would probably prefer if we'd make "intercept" a constant and use that because sometimes in the code I see Intercept with capital "I" and sometimes with lowercase and I think that's error prone and confusing.

For example:
`GLM.java:5073`:
```java
 if ("Intercept".equals(dom[i]))
            throw new IllegalArgumentException("Beta constraints cannot be applied to the Intercept right row.");
```

Reply:
Changed.